code/modules/mob/living/living.dm

```diff
@@ -1029,7 +1029,7 @@
 /mob/living/rad_act(amount)
 	. = ..()
 
-	if(!amount || (amount < RAD_MOB_SKIN_PROTECTION) || HAS_TRAIT(src, TRAIT_RADIMMUNE))
+	if(!amount || (amount < RAD_MOB_SKIN_PROTECTION) || HAS_TRAIT(src, TRAIT_RADIMMUNE) || istype(src, /mob/living/simple_animal/revenant) || istype(src, /mob/living/simple_animal/eminence))
```

Comment:
Give revenant and eminence the rad immune trait, or override rad_act on eminence and revenant since then they will never get rad waves at all and won't be needed in the rad ignore list

Reply:
I think the list is still okay since with it the rad_act will never be called when a checking for wave targets